pom.xml

```diff
@@ -28,18 +36,23 @@
                <Fragment-Host>org.jitsi.libjitsi</Fragment-Host>
                <Bundle-NativeCode>
                 darwin/libjng722.jnilib;osname=Mac OS X;processor=x86-64,
+                darwin/libjnffmpeg.jnilib;osname=Mac OS X;processor=x86-64,
+                darwin/libjnffmpeg-no-openh264.jnilib;osname=Mac OS X;processor=x86-64,
                 linux-x86/libjng722.so;osname=Linux;processor=x86,
                 linux-x86-64/libjng722.so;osname=Linux;processor=x86-64,
                 win32-x86/jng722.dll;osname=Win32;processor=x86,
-                win32-x86-64/jng722.dll;osname=Win32;processor=x86-64
+                win32-x86/jnffmpeg.dll;osname=Win32;processor=x86,
+                win32-x86-64/jng722.dll;osname=Win32;processor=x86-64,
```

Comment:
If all jng722 dlls are deleted, this must be reflected here too. But why were they deleted at all?

Reply:
Current master has the binaries in two places. And this change just removes them from lib where they are not used.

Reply:
Ok, didn't notice that.